mobile/src/modules/auth/screens/AccountsScreen.tsx

```diff
@@ -53,7 +53,9 @@ class Accounts extends React.PureComponent<TProps, State> {
   }
 
   handleAccountPress = (_account: Account) => {
```

Comment:
There is underscore in _account to prevent TS warnings about unused variable. Now there is no need to keep it